packages/sub-modules/routing-table/src/routing-table.ts

```diff
@@ -99,29 +97,20 @@ export class RoutingTable {
   }
 
   /**
-   * Returns an specific node from the routing table based on public key
+   * Returns true or false if the dispatcher url exist on the routing table
    * 
    * @param {URL} url - Node's service url.
-   * @returns {Node} Node object.
+   * @returns {boolean} - True or false.
    * @memberof Routing
    */
-  public getDispatcher(url: URL): Node | Error {
-    const dispatchers = this.store.get(
-      this.dispatchersKey.toUpperCase()
-    ) as URL[];
-    let requestedDispatcher;
-    const urlStr = url.toString();
-
-    dispatchers.forEach(function (storedURL: URL) {
-      if (storedURL.toString() === urlStr) {
-        requestedDispatcher = url;
-      }
-    });
-    if (typeGuard(requestedDispatcher, URL)) {
-      return requestedDispatcher;
-    } else {
-      return new Error("Dispatcher not found in routing table.");
-    }
+  public dispatcherExists(url: URL): boolean {
+    const dispatchers = this.store.get(this.dispatchersKey) as URL[]
+    const urlStr = url.toString()
+
+    // Checking if the dispatcher is already stored
+    const dispatcher = dispatchers.find(element => element.toString() === urlStr)
+
+    return false ? dispatcher === undefined : true
```

Comment:
Ternary operator here is not being used correctly. This way, it will always go to the false path. 

I think this is what we want.
```suggestion
    return dispatcher === undefined ?  false : true
```

or this which is essentially the same,
```suggestion
    return dispatcher ?  true : false
```

Reply:
It's also probably the case that we don't even want this code anymore—we should probably simplify the routing table code as much as we can as we're not using random dispatchers anymore.